blocksuite/blocks/src/root-block/edgeless/frame-manager.ts

```diff
@@ -23,7 +23,11 @@ import type { Store } from '@blocksuite/store';
 import { Text } from '@blocksuite/store';
 import * as Y from 'yjs';
 
-import type { FrameBlockModel, NoteBlockModel } from '../../index.js';
+import {
+  EditPropsStore,
+  type FrameBlockModel,
+  type NoteBlockModel,
+} from '../../index.js';
```

Comment:
For avoiding circular import, `EditPropsStore` should be imported from `@blocksuite/affine-shared/services`